pontoon/base/templates/django/base.html

```diff
@@ -64,7 +64,7 @@
               <div class="menu">
                 <ul>
                   {% if user.is_authenticated %}
-                  <li><a href="{% url 'pontoon.contributors.profile' %}"><i class="fa fa-user fa-fw"></i>My Profile</a></li>
+                  <li><a href="{% url 'pontoon.contributors.contributor.username', username=user %}"><i class="fa fa-user fa-fw"></i>My Profile</a></li>
```

Comment:
Arguments of the [`url` template tag](https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#url) in the Django templates are space-separated values, so you have to remove the comma.